src/wp-admin/js/customize-widgets.js

```diff
@@ -759,7 +759,7 @@
 			/**
 			 * Add the widget reordering elements to the widget
 			 */
-			document.querySelector( '.widget-title-action' ).insertAdjacentHTML( 'beforeend', api.Widgets.data.tpl.widgetReorderNav );
+			this.container[0].querySelector( 'summary' ).insertAdjacentHTML( 'beforeend', api.Widgets.data.tpl.widgetReorderNav );
```

Comment:
Higher in this function there is `var self = this`, would this line be better as:
`self.container[0]...`?

Reply:
And the selector may need reverting also.

Reply:
I agree about changing `this` to `self`. But I think the selector is correct. The file we inherited from WP originally had:
```
this.container.find( '.widget-title-action' ).after( $( api.Widgets.data.tpl.widgetReorderNav ) );
```
I've used `insertAdjacentHTML` as the method of inserting the HTML, but that's not a direct equivalent to jQuery's `after`. It's more like `append` (except that it's for inserting a string and not an element). So that necessitates a change of selector.